pkg/drivers/kic/oci/network_create.go

```diff
@@ -75,6 +75,28 @@ func CreateNetwork(ociBin string, networkName string) (net.IP, error) {
 	// Rather than iterate through all of the valid subnets, give up at 20 to avoid a lengthy user delay for something that is unlikely to work.
 	// will be like 192.168.49.0/24 ,...,192.168.239.0/24
 	for attempts < 20 {
+		// check if subnetAddr overlaps with *any* existing local networks
```

Comment:
thanks for fixing this, do u mind sharing an example before after this PR that shows this fixes it ?
@prezha 
could we add the same logic to KVM network create as well ? 
so docker already existed, and KVM tried to start in the network that is overlapping, it creates in separate network?

Reply:
@medyagh, sure: i've added same logic for kvm as well and amended pr details with an example with before/after and some notes